docs/pages/versions/unversioned/sdk/prevent-screen-shot.md

```diff
@@ -0,0 +1,95 @@
+---
+title: PreventScreenShot
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-38/packages/expo-prevent-screen-shot'
```

Comment:
Should `tree/…/packages` point to a specific SDK version? This seems like a strange thing.

Reply:
in the `unversioned` docs, this doesn't matter too much, since it's used for the link at the bottom of the API pages "See source code for this module". 

Outside of scope- right now we don't automate this when we generate the new API docs, but we should because a lot of API docs point to an older version's source code (eg - https://github.com/expo/expo/blob/master/docs/pages/versions/v37.0.0/sdk/accelerometer.md)
Will add that to my todos

Reply:
Thanks for the explanation and thank you for taking care of the automation!